packages/ruleset/src/functions/discriminator-property-exists.js

```diff
@@ -53,8 +53,7 @@ function validateDiscriminators(schema, path) {
   if (!schemaHasProperty(schema, propertyName)) {
     logger.debug(`Discriminator property is not defined: ${propertyName}`);
     errors.push({
-      message:
-        'The discriminator property name used must be defined in this schema',
+      message: `The discriminator property must be defined in this schema: ${propertyName}`,
```

Comment:
To me, this could read like the variable at the end is the schema name ("this schema: ..."). I don't want to twist the sentence around too much but perhaps this would be better? Idk. Take or leave this suggestion 🙂 

```suggestion
      message: `Discriminator property must be defined in the schema: ${propertyName}`,
```